components/pipeline-service/development/kustomization.yaml

```diff
@@ -8,55 +8,6 @@ commonAnnotations:
   argocd.argoproj.io/sync-options: SkipDryRunOnMissingResource=true
 
 resources:
-  - github.com/minio/operator?ref=v5.0.15
```

Comment:
I remember this was recently changed by @amisstea , so I wonder if his team has any use of it. @amisstea please confirm we can remove that.